From 610b1e271413e627e41b171ad22f77c29a1864c1 Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Wed, 9 Mar 2022 17:09:27 +0100
Subject: [PATCH] fmt: hack to compile with fmt 8.1

quick and dirty hack to avoid compile error with fmt 8.1 until properly fixed

https://github.com/xbmc/xbmc/issues/20789
---
 xbmc/rendering/gl/RenderSystemGL.cpp     | 2 +-
 xbmc/rendering/gles/RenderSystemGLES.cpp | 2 +-
 xbmc/storage/MediaManager.cpp            | 4 ++--
 3 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/xbmc/rendering/gl/RenderSystemGL.cpp b/xbmc/rendering/gl/RenderSystemGL.cpp
index ebcac56e25f..e0bf9a8b9b4 100644
--- a/xbmc/rendering/gl/RenderSystemGL.cpp
+++ b/xbmc/rendering/gl/RenderSystemGL.cpp
@@ -719,7 +719,7 @@ void CRenderSystemGL::EnableShader(ShaderMethodGL method)
   }
   else
   {
-    CLog::Log(LOGERROR, "Invalid GUI Shader selected {}", method);
+    // CLog::Log(LOGERROR, "Invalid GUI Shader selected {}", method);
   }
 }
 
diff --git a/xbmc/rendering/gles/RenderSystemGLES.cpp b/xbmc/rendering/gles/RenderSystemGLES.cpp
index d8557ff8a0f..68ab6e961cd 100644
--- a/xbmc/rendering/gles/RenderSystemGLES.cpp
+++ b/xbmc/rendering/gles/RenderSystemGLES.cpp
@@ -562,7 +562,7 @@ void CRenderSystemGLES::EnableGUIShader(ShaderMethodGLES method)
   }
   else
   {
-    CLog::Log(LOGERROR, "Invalid GUI Shader selected - {}", method);
+//    CLog::Log(LOGERROR, "Invalid GUI Shader selected - {}", method);
   }
 }
 
diff --git a/xbmc/storage/MediaManager.cpp b/xbmc/storage/MediaManager.cpp
index 4bd5d9d6e21..5db22765d44 100644
--- a/xbmc/storage/MediaManager.cpp
+++ b/xbmc/storage/MediaManager.cpp
@@ -560,8 +560,8 @@ std::string CMediaManager::GetDiskUniqueId(const std::string& devicePath)
   }
 
   std::string strID = StringUtils::Format("removable://{}_{}", info.name, info.serial);
-  CLog::Log(LOGDEBUG, "GetDiskUniqueId: Got ID {} for {} with path {}", strID, info.type,
-            CURL::GetRedacted(mediaPath));
+  // CLog::Log(LOGDEBUG, "GetDiskUniqueId: Got ID {} for {} with path {}", strID, info.type,
+            // CURL::GetRedacted(mediaPath));
 
   return strID;
 }
-- 
2.25.1

